bbp/modules/default.nix

```diff
@@ -781,6 +781,17 @@ let
             conflicts = conflicts-modules;
         };
 
+        snakemake = pkgs.envModuleGen rec {
+            name = "snakemake";
+            moduleFilePrefix = "nix/py36";
```

Comment:
Should I better place it to 'nix/nse', as probably not a general usage tool?

Reply:
Yes, let's keep it consistent

Reply:
But if it is the case, move it under bbp/nse packages g roup and not additionalPythonPackages

Reply:
As it is a third-party tool, I felt reluctant to put it to bbp/nse.
But if bbp/nse is not intended to mean the "authorship", I'll put `snakemake` there (leaving it's library dependencies in additionalPythonPackages?).

Reply:
On the other hand, things like `pillow` or `keras` appear to be pretty specific as well (at least not as general as `ipython & Co`).
If it does not make any difference, let's keep `snakemake` at `nix/py36`, that's how I see the consistency here (might be wrong).

Reply:
The separation I did was not about specificity, but about "internal vs external".

If a package is developped internally -> bbp
If a package comes from the outside -> patches

The only exceptions I did are about package that became "mainstream" from BBP and you might mind in third party software repositories ( Official fedora packages,  official PYPI, or other) that I put in patches.

Reply:
OK, `snakemake` comes from the outside.
Then I guess we don't have to change anything here (package should live in additionalPackages, module in `nix/py36` namespace)?

Reply:
yep